SU-HSDO/suhumsci
# feat(shs-5076): adding additional image styles to hero no text

# READY FOR REVIEW

## Summary
This PR adds some larger image styles to the Hero No Text responsive image style for banner images

## Need Review By (Date)
asap

## Urgency
medium

## Steps to Test
1. Add a banner image to a test page (the Full-width Region of a Flexible Page)
2. Check that images are using the new styles (the largest new image preset is 3000x750px).

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
